homeassistant/components/lutron/strings.json

```diff
@@ -0,0 +1,35 @@
+{
+  "config": {
+    "error": {
+      "cannot_connect": "[%key:common::config_flow::error::cannot_connect%]",
+      "unknown": "[%key:common::config_flow::error::unknown%]"
+    },
+    "step": {
+      "user": {
+        "data": {
+          "host": "[%key:common::config_flow::data::host%]",
+          "username": "[%key:common::config_flow::data::username%]",
+          "password": "[%key:common::config_flow::data::password%]"
+        },
+        "data_description": {
+          "host": "The hostname or IP address of the Lutron main repeater."
+        },
+        "description": "Please enter the main repeater login information",
+        "title": "Main repeater setup"
+      }
+    },
+    "abort": {
+      "single_instance_allowed": "[%key:common::config_flow::abort::single_instance_allowed%]"
+    }
+  },
+  "issues": {
+    "deprecated_yaml_import_issue_cannot_connect": {
+      "title": "The Lutron YAML configuration import cannot connect to server",
+      "description": "Configuring Lutron using YAML is being removed but there was an connection error importing your YAML configuration.\n\nMake sure your home assistant can reach the main repeater. In case the main repeater is down, try again later."
+    },
+    "deprecated_yaml_import_issue_unknown": {
+      "title": "The Lutron YAML configuration import request failed due to an unknown error",
+      "description": "Configuring Lutron using YAML is being removed but there was an unknown error while importing your existing configuration.\n\nSetup will proceed with required configuration."
```

Comment:
This is not a great description. If this happens, what should the user do? What does `Setup will proceed with required configuration` mean?

Reply:
updated to add clarity